sql/mozfun/bytes/zero_right/metadata.yaml

```diff
@@ -1,2 +1,10 @@
-description: Zero bits on the right of byte
```

Comment:
This seems to be `metadata.yaml` format for defining query settings and scheduling. Here's an example for a UDF: 
https://github.com/mozilla/bigquery-etl/blob/main/sql/mozfun/bits28/active_in_range/metadata.yaml

Reply:
Also, UDF definition seems to be missing in this PR. Not sure if this is intentional?

Reply:
Ah, I see you're added a missing `metadata.yaml` for the UDF rather than adding a new UDF.

Reply:
Thanks for adding it.